src/app/traveler/traveler.component.html

```diff
@@ -1,54 +1,61 @@
+<div id="traveler-sidebar" class="active">
+  <ul id="traveler-sidebar-list">   
+    <li id="traveler-side-top" class="traveler-logout">
+      <button id="traveler-logout-button" type="button" class="btn btn-primary" (click)="toggleFlights()">
+        {{flightButtonText}}
+      </button>
+    </li>
+    <li class="traveler-logout">
+      <button type="button" class="btn btn-danger" id="traveler-logout-button"(click)="openLogoutModal(logoutTravelerModal)">
```

Comment:
I tried out a danger button for the logout but I don't think it's a great fit for the mint background of the sidebar.

Reply:
matched your button classes for consistency